src/plugins/ui/evaluation_chart.ts

```diff
@@ -61,20 +66,31 @@ export class EvaluationChartPlugin extends UIPlugin {
    *
    *  - the chart background color if one is defined
    *  - the fill color of the cell if one is defined
+   *  - the text color of the cell if one is defined
    *  - the fill color of the cell from conditional formats if one is defined
+   *  - the text color of the cell from conditional formats if one is defined
    *  - the default chart color if no other color is defined
+   *  - the default text color if no other color is defined
    */
-  getBackgroundOfSingleCellChart(
+  getStyleOfSingleCellChart(
```

Comment:
the docstring first line can be updated :wink: It's no longer `Get the background color of...`

Also, the docstring is actually wrong (since the beginning ?). The CF style takes precedence over cell defined style.
I think the all this "order of priority" part can be removed. It's doomed to be outdated at some point. If someone wants to know how it's computed, they go and check `getCellComputedStyle`

Reply:
Gotcha! Removed them :+1: